gui/wxpython/main_window/frame.py

```diff
@@ -637,16 +636,21 @@ def BuildPanes(self):
             .Name("python")
             .Caption("Python")
             .Right()
-            .Layer(2)
-            .Position(3)
             .BestSize(self.PANE_BEST_SIZE)
             .MinSize(self.PANE_MIN_SIZE)
             .CloseButton(False)
             .MinimizeButton(True)
             .MaximizeButton(True),
+            target=self._auimgr.GetPane("modules"),
         )
 
         self._auimgr.GetPane("toolbarNviz").Hide()
+
+        # Set Modules as active tab
+        modules = self._auimgr.GetPane("modules")
+        notebook = self._auimgr.GetNotebooks()[0]
```

Comment:
I was playing with the sizing and this is the only thing that worked:
```
+        pane = self._auimgr.GetPane(notebook)
+        pane.MinSize(self.goutput.GetMinSize())
```

Reply:
Yes, it works, thank you Anicka.

Reply:
> I have found out why it displays invalid map size 0,413. The problem is in the function ChangeMapSize which is called several times each time with GetClientSize() parameter. And this parameter is being changed. I tried to write down the process with "GetClientSize" during the startup:
> 
> OnIdle
> (0, 413)
> Invalid map size 0,413
> _updateM
> (569, 421)
> OnIdle
> (569, 421)
> _updateM
> (569, 421)
> 
> The problem is in the first call of OnIdle, then it is alright...
> But I think that the problem is generally caused by very small map size.
> When we set the size of notebook based on Modules tab, it is alright. And it also alright in terms of GTKCheck warnings in cmd.
> So, I would propose to use Modules tab for this purpose and do not deal with render.py. What do you think @petrasovaa ?

Sorry, fault. Those warnings are still displayed:
(wxgui.py:10746): Gtk-WARNING **: 00:04:09.085: Negative content width -6 (allocation 4, extents 5x5) while allocating gadget (node checkbutton, owner GtkCheckButton)

(wxgui.py:10746): Gtk-CRITICAL **: 00:04:09.085: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkCheckButton

(wxgui.py:10746): Gtk-WARNING **: 00:04:09.156: Negative content width -6 (allocation 4, extents 5x5) while allocating gadget (node checkbutton, owner GtkCheckButton)

(wxgui.py:10746): Gtk-CRITICAL **: 00:04:09.157: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkCheckButton

if we used python tab, they would not be displayed, but then the notebook is too narrow...

I tried also the thing with BestSize but it does not work - notebook is shrunk (same as the second picture in this PR).